A local nix-review of the whole PR ended with:
```shell
builder for '/nix/store/c607qhmbxh1x5w9nackkkvfddqalf1a5-python3.7-python-efl-1.21.0.drv' failed with exit code 1; last 10 log lines:
  efl/eo/efl.eo.c: In function ‘__pyx_pf_3efl_2eo_8event_global_thaw’:
  efl/eo/efl.eo.c:1984:3: error: too many arguments to function ‘efl_event_global_thaw’
     efl_event_global_thaw(((Eo const *)efl_object_class_get()));
     ^~~~~~~~~~~~~~~~~~~~~
  In file included from /nix/store/p4z6kzani0zfcqdrnlya699hgr5hlkm7-efl-1.22.1/include/eo-1/Eo.h:216:0,
                   from efl/eo/efl.eo.c:614:
  /nix/store/p4z6kzani0zfcqdrnlya699hgr5hlkm7-efl-1.22.1/include/eo-1/efl_object.eo.h:511:12: note: declared here
   EOAPI void efl_event_global_thaw(void);
              ^~~~~~~~~~~~~~~~~~~~~
  error: command 'gcc' failed with exit status 1
[...]
3 package failed to build:
enlightenment.econnman python27Packages.pythonefl python37Packages.pythonefl
```

Let's wait for ofBorg's build results

--- a/pkgs/development/python-modules/python-efl/default.nix
+++ b/pkgs/development/python-modules/python-efl/default.nix
@@ -4,11 +4,11 @@
 
 buildPythonPackage rec {
   name = "python-efl-${version}";
-  version = "1.21.0";
+  version = "1.22.0";
 
   src = fetchurl {
     url = "http://download.enlightenment.org/rel/bindings/python/${name}.tar.xz";
-    sha256 = "08x2cv8hnf004c3711250wrax21ffj5y8951pvk77h98als4pq47";
+    sha256 = "1qhy63c3fs2bxkx2np5z14hyxbr12ii030crsjnhpbyw3mic0s63";
   };
 
   nativeBuildInputs = [ pkgconfig ];

